specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/billing.json

```diff
@@ -0,0 +1,1371 @@
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Operation": {
+      "description": "A Billing REST API operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name: {provider}/{resource}/{operation}.",
+          "type": "string",
+          "readOnly": true
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.Billing.",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Invoice, etc.",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result listing billing operations. It contains a list of operations and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "description": "List of billing operations supported by the Microsoft.Billing resource provider.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Resource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "initiateImportRequest": {
+      "type": "object",
+      "description": "Request parameters to initiate import request.",
+      "properties": {
+        "properties": {
+          "description": "Request parameters to initiate import request.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/initiateImportRequestProperties"
+        }
+      }
+    },
+    "initiateImportRequestProperties": {
+      "type": "object",
+      "description": "Request parameters to initiate import request.",
+      "properties": {
+        "devTestUsageContext": {
```

Comment:
Does the user understand what a "context" is? Is this an arbitrary string or is some server-side validation occur?

Reply:
User understands what context is. Also there are few validations for it on server side.